tests/parse/test_parse_vrplib.py

```diff
@@ -231,6 +231,25 @@ def test_parse_vrplib_do_not_compute_edge_weights():
     assert_("edge_weight" not in actual)
 
 
+def test_parse_vrplib_raises_data_specification_and_section():
+    """
+    Tests that a ValueError is raised when data is included both as
+    specification and section.
+    """
+    instance = "\n".join(
+        [
+            "SERVICE_TIME: 10",
+            "SERVICE_TIME_SECTION",
+            "1    20",
+            "2    20",
```

Comment:
What happens when the section comes before the specification? Does that also raise?

Reply:
That raises an error, but it's not related to sections coming before specifications. I think we should be strict and add that too. I'll do that in another PR

Reply:
Yeah that works